content/_code-samples/tx-serialization/js/test-cases/README.md

```diff
@@ -0,0 +1,52 @@
+# Transaction Serialization Test Cases
+
+This folder contains several transactions in their JSON and binary forms, which
+you can use to verify the behavior of the transaction serialization code.
+
+For example (starting from the `tx-serialization/js/` dir above this one):
+
+```bash
+$ node index.js -f test-cases/tx2.json | \
+  diff - test-cases/tx2-binary.txt
```

Comment:
Unfortunately this instruction is out of date now because the pretty print. Could you update the expected result to match the pretty print version? (Maybe renaming the files to `tx2-expected.txt` would better convey what they contain if you do)

Reply:
I did it by adding a --raw CLI parameter for such cases, it's reflected in the README.md